tools/yarn_install.R

```diff
@@ -46,13 +46,34 @@ needs_prefix <- c(
   "backface-visibility", "touch-action"
 )
 for (prop in needs_prefix) {
-  scss_src <- lapply(scss_src, add_prefixes, prop)
+  scss_src <- lapply(scss_src, add_property_prefixes, prop)
 }
 # Conditionally prefix text-decoration if its not CSS2 compliant
 # https://www.w3.org/TR/CSS2/text.html#lining-striking-props
 # https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration
 # https://caniuse.com/#search=text-decoration
-scss_src <- lapply(scss_src, add_prefixes, "text-decoration", c("none", "underline", "overline", "line-through", "blink", "inherit"))
+scss_src <- lapply(
+  scss_src, add_property_prefixes, "text-decoration",
+  c("none", "underline", "overline", "line-through", "blink", "inherit")
+)
+
+
+add_value_prefixes <- function(src, value) {
+  pattern <- paste0("^\\s*(.+):\\s*", value)
+  idx <- grep(pattern, src)
+  for (i in idx) {
+    prop_val <- strsplit(src[[i]], ":\\s*")[[1]]
+    src[[i]] <- paste0(
+      prop_val[1], ": ", c("-webkit-", "-moz-", "-ms-"), prop_val[2],
```

Comment:
http://thenewcode.com/217/CSS3-Vendor-Prefixes
> As a rule, vendor prefixed properties are written first in any style declaration, with the final version of the property at the end:
>
> ```css
> div#mainbox {
>	-moz-border-radius: 5px; 
> 	-webkit-border-radius: 5px; 
> 	border-radius: 5px;
> }
> ```

```suggestion
      prop_val[1], ": ", c("-webkit-", "-moz-", "-ms-", "-o", ""), prop_val[2],
```